LilitYeghiazaryan/functions_12

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+void func_3(double a, double b, double*& res)
```

Comment:
@lilitYegh I don't think you need a reference (`&`) here. BTW, use `vector` instead otherwise your current code has a memory leak (you don't delete the array)

Reply:
Ok. I'll try to change it.

> @lilitYegh I don't think you need a reference (`&`) here. BTW, use `vector` instead otherwise your current code has a memory leak (you don't delete the array)

Ok. I'll try to change it.